usaspending_api/references/management/commands/load_gtas.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connections, transaction
+
+from usaspending_api.etl.broker_etl_helpers import dictfetchall
+from usaspending_api.references.models import GTASTotalObligation
+
+logger = logging.getLogger('console')
+
+TOTAL_OBLIGATION_SQL = '''
+SELECT
+    fiscal_year,
+    CASE
+        WHEN sf_133.period = 3 THEN 1
+        WHEN sf_133.period = 6 THEN 2
+        WHEN sf_133.period = 9 THEN 3
+        WHEN sf_133.period = 12 THEN 4
```

Comment:
are these values (3,6,9,12) the only allowed values for that sf_133.period field?

Reply:
yes, that's correct. These values are cumulative so we want the latest, cumulative value for each quarter, which equates to the last fiscal period for each fiscal quarter.